utils/s2n_fork_detection.c

```diff
@@ -118,6 +118,7 @@ static inline S2N_RESULT s2n_initialise_inherit_zero(void *addr, long page_size)
     return S2N_RESULT_OK;
 }
 
+#if !defined (__OpenBSD__)
```

Comment:
I'm guessing that this ifdef is because the compiler complains about the method being unused? Maybe we should just remove the "static" to reduce our conditional compilation.

Reply:
Correct. I've removed `static` and the `#if`.